app/Shell/Docker.php

```diff
@@ -120,7 +120,9 @@ public function bootContainer(string $dockerRunTemplate, array $parameters): voi
         $process = $this->shell->exec('docker run -d --name "${:container_name}" ' . $dockerRunTemplate, $parameters);
 
         if (! $process->isSuccessful()) {
-            throw new Exception("Failed installing {$containerName}");
+            $containerName = array_key_exists('container_name', $parameters) ? $parameters['container_name'] : '';
+            $imageName = $parameters['image_name'];
+            throw new Exception("Failed installing {$imageName}. The container {$containerName} is already in use.");
```

Comment:
```suggestion
            throw new Exception("Failed installing {$parameters['imageName']}.");
```

Reply:
It's more generic. I can change my PR if you want 😃